OPHD/ShellOpenPath.cpp

```diff
@@ -21,6 +22,10 @@ namespace
 
 void shellOpenPath(const std::string& path)
 {
-	// Explicitly ignore implementation defined return value
-	std::ignore = std::system((std::string{ShellOpenCommand} + " " + path).c_str());
+	#if defined(_WIN32)
+		std::ignore = ShellExecuteA(NULL, "explore", path.c_str(), NULL, NULL, SW_NORMAL);
```

Comment:
This could probably be improved so it's not hardcoded to use `ShellExecuteA`. Though we do have a hardcoded `MessageBoxA` already.

You may want to test this, as I'm not certain the encoding being passed will match what's expected on Windows.

Reply:
ShellExecute (at least in visual studio) is used as an alias between ShellExecuteA and ShellExecuteW -- on my system ShellExecuteW is what it expands to in order to handle unicode paths. It appears to work on this end, and is used as described in MS's documentation. Someone on a non-US English rig would provide a good test case.